packages/flutter/lib/src/material/expand_icon.dart

```diff
@@ -59,6 +60,11 @@ class ExpandIcon extends StatefulWidget {
   /// This property must not be null. It defaults to 8.0 padding on all sides.
   final EdgeInsetsGeometry padding;
 
+  /// The color of the icon.
+  ///
+  /// It defaults with theme.brightness.
```

Comment:
It would be more helpful to be explicit: 
The default color is [Colors.white54] for dark themes, [Colors.black54] for light themes. The [Theme]'s brightness is specified by [ThemeData.brightness].